bluepyemodel/model/model_configurator.py

```diff
@@ -144,13 +144,21 @@ def get_gene_based_configuration(self):
             version = None
             for k in nexus_keys:
```

Comment:
I don't really get what nexus_keys is supposed to be here. From reading the icselector code, it looks like it is always an empty dict

Reply:
From my understanding, it seems that it contains a dictionary with {name, modelid} entries for each mechanism that has a nexus entry in the ion channel mapping json file